src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -86,7 +88,7 @@ public function normalize($object, $format = null, array $context = array())
         $data = array();
         $stack = array();
         $attributes = $this->getAttributes($object, $format, $context);
-        $class = \get_class($object);
+        $class = $this->objectClassResolver ? \call_user_func($this->objectClassResolver, $object) : \get_class($object);
```

Comment:
could be written as `($this->objectClassResolver)($object)`

Reply:
I've copied what was done for the `maxDepthHandler`.
https://github.com/symfony/symfony/blob/9610d10034223d031d015bb92488d20040ad1725/src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php#L100
It seems to me there is no important difference between the two ways of doing it: https://stackoverflow.com/questions/1596221/php-call-user-func-vs-just-calling-function/35031466

Reply:
`($this->objectClassResolver)($object)` is more elegant, but indeed being consistent is good